environment.js

```diff
@@ -0,0 +1,43 @@
+// Fallback to rinkeby
+const FALLBACK_CHAIN_ID = 4
+
+const ENV_VARS = {
+  BUILD() {
+    return process.env.BUILD || 'undefined'
+  },
+  DEFAULT_CHAIN_ID() {
```

Comment:
This is defined as "default" because we plan to provide dynamic network config based on optional url param.